.github/workflows/plugin-packages.yml

```diff
@@ -102,7 +99,7 @@ jobs:
         go-version: 1.15
 
     - name: Authenticate and Connect to GKE cluster
-      if: (needs.pre-build.outputs.testLogCollector == 'true') || startsWith(github.ref, 'refs/tags/')
```

Comment:
Do not remove this line... instead replace `testLogCollector` with  `testPreflight` here

Reply:
resolved